pkg/controller/usersignup/usersignup_controller_test.go

```diff
@@ -594,6 +604,77 @@ func TestUserSignupWithExistingMUROK(t *testing.T) {
 		}
 	}
 
+	require.Equal(t, mur.Name, instance.Status.CompliantUsername)
+	require.NotNil(t, cond)
+	require.Equal(t, v1.ConditionTrue, cond.Status)
```

Comment:
You can use https://github.com/codeready-toolchain/toolchain-common/blob/master/pkg/test/condition.go#L26 instead of manually look up the condition.